app/forms/date_of_birth_form.rb

```diff
@@ -0,0 +1,81 @@
+class DateOfBirthForm < Form
```

Comment:
If we merge https://github.com/DFE-Digital/claim-additional-payments-for-teaching/pull/4009
Most of this could be replaced with

```
include DateOfBirth
self.date_of_birth_field = :date_of_birth
```